products/sourcerepo/terraform.yaml

```diff
@@ -26,13 +26,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_name: "fmt.Sprintf(\"my-repository%s\", context[\"random_suffix\"])"
         vars:
           repository_name: "my-repository"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sourcerepo_repository_full"
+        primary_resource_id: "my-repo"
+        primary_resource_name: "fmt.Sprintf(\"my-repository%s\", context[\"random_suffix\"])"
```

Comment:
You don't need to manually append this suffix here. The purpose of assigning values to fields here rather than in the `.tf.erb` file is that the generation will append the suffix for you.